.gitignore

```diff
@@ -1,2 +1,2 @@
 .DS_Store
-node_modules/
\ No newline at end of file
+node_modules/package-lock.json
```

Comment:
It looks like this file didn't have a trailing newline, so the command to append to this file didn't work right. Can you add a newline between `node_modules/` and `package-lock.json`?